Project.toml

```
@@ -1,7 +1,14 @@
 name = "IceFloeTracker"
 uuid = "04643c7a-9ac6-48c5-822f-2704f9e70bd3"
-authors = ["bradford roarr <bradford_roarr@brown.edu> and contributors"]
-version = "0.6.2"
+authors = [
+    "Carlos Paniagua <carlos_paniagua@brown.edu>",
+    "Timothy Divoll <bradford_roarr@brown.edu>",
+    "John Gerrard Holland <john_holland1@brown.edu>",
+    "bradford roarr <bradford_roarr@brown.edu>",
```

Name-case for Brad's name?
```suggestion
    "Bradford Roarr <bradford_roarr@brown.edu>",
```

---

@broarr, any preference?

---

Good catch, definitely put all the names in the same format. I figure that if Brad had a super strong opinion about not using capital letters, it would be in lowercase on the CCV People page. I'd say capitalize it now and we can fix the format later if there's a different form of the name wanted.

---

No preference on case on my name